generators/entity-client/templates/react/src/main/webapp/app/entities/entity-update.tsx.ejs

```diff
@@ -227,11 +227,17 @@ export const <%= entityReactName %>Update = (props: I<%= entityReactName %>Updat
       <Col md="8">
         { loading ? <p>Loading...</p> :
         <AvForm model={isNew ? {} : <%= entityInstance %>Entity} onSubmit={saveEntity} >
-          <%_ for (field of fields) {
+          { !isNew ?
+            <AvGroup>
+              <Label for="<%= entityFileName %>-id"><Translate contentKey="global.field.id">ID</Translate></Label>
+              <AvInput id="<%= entityFileName %>-id" type="text" className="form-control" name="id" required readOnly/>
```

Comment:
My bad, I took deeper look and the fix should be something like:
```suggestion
              <Label for="<%= entityFileName %>-<%= primaryKey.fields[0].fieldName %>"><Translate contentKey="<%= primaryKey.fields[0].fieldTranslationKey %>"><%= primaryKey.fields[0].fieldNameHumanized %></Translate></Label>
              <AvInput id="<%= entityFileName %>-<%= primaryKey.fields[0].fieldName %>" type="text" className="form-control" name="<%= primaryKey.fields[0].fieldName %>" required readOnly/>
```
`fieldTranslationKey` and `fieldNameHumanized` are not available directly in the primaryKey.